apps/api-extractor-model/src/model/ApiIndexSignature.ts

```diff
@@ -59,4 +60,12 @@ export class ApiIndexSignature extends ApiParameterListMixin(ApiReleaseTagMixin(
   public get containerKey(): string {
     return ApiIndexSignature.getContainerKey(this.overloadIndex);
   }
+
+  /** @beta @override */
+  public buildCanonicalReference(): DeclarationReference {
+    return (this.parent ? this.parent.canonicalReference : DeclarationReference.empty())
+      .addNavigationStep(Navigation.Members, '')
```

Comment:
You either need a name here, or we need to specialize `:signature` and remove the navigation step.